### ISSUE TITLE
Suggestion for liked button functionality

### ISSUE THREAD

**Issue by AlexDeFoc** [futo-org/grayjay-android#1031]
So when we hit the like button on a video, it's for nothing because we cannot find the videos that are being liked.

I propose that GrayJay would come with a predefined playlist just like the watch later playlist or if it's not that hard the used when they first hit the like button to assign to a certain playlist.
And so this is the functionality : because we can "save to a playlist" a video or add to watch later which they just add the video to a playlist, why cannot the like button just add the video to a playlist.

TL;DR : Make like button add video to a playlist, and liking it again to remove the like button, remove the video from that certain playlist.

This seems very simple to add and can be just a barebones buggy feature to be added as fast as posible.

My biggest disadvantage for GrayJay is that when i like a video i gotta add it manually to a certain playlist.
